hack/.ci/manifests/cluster/nodeconfig-openshift-aws.yaml

```diff
@@ -17,7 +17,8 @@ spec:
       type: RAID0
       RAID0:
         devices:
-          nameRegex: ^/dev/nvme[1-9]\d*n\d+$
+          modelRegex: Amazon EC2 NVMe Instance Storage
+          nameRegex: ^/dev/nvme\d+n\d+$
```

Comment:
```suggestion
          nameRegex: '^/dev/nvme\d+n\d+$'
```

Reply:
it's neither required nor consistent with other yamls. I'll leave it as is.

Reply:
Sorry - this comment was half in writing and I accidentally sent it. Yes of course.